etstool.py

```diff
@@ -176,17 +175,15 @@ def install(edm, runtime, environment, editable, docs, source):
         "{edm} --config edm.yaml install -y -e {environment} " + packages,
         "{edm} plumbing remove-package -e {environment} traits",
     ]
-    if editable:
-        install_cmd = (
-            "{edm} run -e {environment} -- "
-            "python -m pip install --editable . --no-dependencies"
-        )
-    else:
-        install_cmd = (
-            "{edm} run -e {environment} -- "
-            "python -m pip install . --no-dependencies"
-        )
+
+    install_cmd = _get_install_command_string(".", editable)
+    install_mypy_cmd = _get_install_command_string("mypy", editable, False)
```

Comment:
Let's follow the normal Python passing conventions here and pass the optional argument by name. (That helps readers of the code: it's not obvious what the `False` is doing otherwise.)

Reply:
This command breaks the editable install: `python etstool.py install --editable` complains that:
```
ERROR: mypy is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).
```
We want to pass `editable=False` for the mypy install.

Reply:
> I'll create an issue.

Done: https://github.com/enthought/buildsystem/issues/2984

Reply:
Done